### PR TITLE
Sg/exposed property ui enhancements

### PR DESCRIPTION
**Pull request by martint-unity**
### Purpose of this PR
Showing that a property is exposed with a green circle

---
### Release Notes
Showing that a property is exposed with a green circle

---
### Testing status
Running Katana
Passed 
https://katana.bf.unity3d.com/projects/com.unity.render-pipelines/builders/proj57-All%20Tests/builds/1201?automation-tools_branch=add-platform-filter&ScriptableRenderLoop_branch=sg/exposed-property-ui-enhancements&unity_branch=trunk

---
### Overall Product Risks
**Technical Risk**: None, Low, Medium, High?
None
**Halo Effect**: None, Low, Medium, High?
None
---
### Comments to reviewers
Notes for the reviewers you have assigned.

### CODE REVIEW

File: com.unity.shadergraph/CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [5.1.0-preview] - 2018-10-18
+### Fixed
+- When a property is exposed in the blackboard, it will have a small green circle on it.
```
Comment (pbbastian) [inline com.unity.shadergraph/CHANGELOG.md]: The green dot has always been there, so I think this is approaching it from the wrong direction. Suggestion:
> Non-exposed properties in the blackboard no longer have a green dot next to them.

File: com.unity.shadergraph/Editor/Drawing/Blackboard/BlackboardProvider.cs [hunk 2]
```diff
@@ -238,8 +237,8 @@ void AddProperty(IShaderProperty property, bool create = false, int index = -1)
             if (create)
                 property.displayName = m_Graph.SanitizePropertyName(property.displayName);
 
-            var field = new BlackboardField(m_ExposedIcon, property.displayName, property.propertyType.ToString()) { userData = property };
-            var row = new BlackboardRow(field, new BlackboardFieldPropertyView(m_Graph, property));
+            var field = new BlackboardField(exposedIcon, property.displayName, property.propertyType.ToString()) { userData = property };
```
Comment (pbbastian) [inline com.unity.shadergraph/Editor/Drawing/Blackboard/BlackboardProvider.cs]: replace `exposedIcon` with `property.generatePropertyBlock != null ? exposedIcon : null`

File: com.unity.shadergraph/CHANGELOG.md (hunk 1 above)
Comment (Katstodian) [reply com.unity.shadergraph/CHANGELOG.md]: I approve of Peter's suggestion.

**Comment by martint-unity** [comment]
Hello :)

**Pull request merged**